api/src/main/scala/uk/ac/wellcome/platform/api/responses/Responses.scala

```diff
@@ -14,7 +18,64 @@ case class ResultListResponse(
   pageSize: Int = 10,
   totalPages: Int = 10,
   totalResults: Int = 100,
-  results: Array[_ <: Any]
+  results: Array[_ <: Any],
+  prev: Option[String] = None,
+  next: Option[String] = None
 ) {
   @JsonProperty("type") val ontologyType: String = "ResultList"
 }
+
+object ResultListResponse {
+
+  private def getParamMapFromCustomCCRequestParams(cc: ApiRequest): ParamMap =
+    ParamMap((Map[String, String]() /: cc.getClass.getDeclaredFields) {(a, f) =>
+      f.setAccessible(true)
+
+      if(f.getName != "request") {
```

Comment:
I'm aware this might be better off as a comment actually in the code. How non-obvious is this do we think?